llpc/test/shaderdb/partial_pipeline_cache_test/partial_pipeline_cache_test.ll

```diff
@@ -0,0 +1,11 @@
+; Test that partial pipeline cache is handled correctly.
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v -shader-cache-mode=1 %S/pipelines/*.pipe | FileCheck -check-prefix=SHADERTEST %s
```

Comment:
I think it would be better to be more explicit and list all `.pipe` files instead of relying on the shell expansion.

Reply:
Fixed.